bot/slack/commands/roulette.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+
+class command():
```

Comment:
As before, class names should be in `PascalCase`.

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1018/commits/458723466763d1f51f2879147d4c1c255c4ae221